pkg/cmd/server/api/register.go

```diff
@@ -3,10 +3,13 @@ package api
 import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/runtime/serializer"
 )
 
 var Scheme = runtime.NewScheme()
 
+var Codecs = serializer.NewCodecFactory(Scheme)
```

Comment:
Currently yes, but it should probably be used elsewhere to mirror upstream: https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/register.go#L30